src/views/Contacts.vue

```diff
@@ -193,6 +196,8 @@ export default {
 				return this.sortedContacts
 			} else if (this.selectedGroup === GROUP_NO_GROUP_CONTACTS) {
 				return this.ungroupedContacts.map(contact => this.sortedContacts.find(item => item.key === contact.key))
+			} else if (this.selectedGroup === ROUTE_CIRCLE) {
+				return this.circles
```

Comment:
I think this too hacky, because you're returning circles to contact lists when ContactList should be empty. 
I think a better approach would be to and a condition here 
https://github.com/nextcloud/contacts/blob/fix/circlesRouting/src/components/AppNavigation/Settings/SettingsImportContacts.vue#L22